<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISOI Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Control</string>
     </attribute>
     <widget class="QSlider" name="expSlider">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>160</width>
        <height>19</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="liveBtn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>LIVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cropBtn">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Crop</string>
      </property>
     </widget>
     <widget class="QLabel" name="expLabel">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Exposure :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="histoBtn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Histogram</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Camera settings</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="C_expSb">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LEDs</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Green">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>75</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>GREEN</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Red">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>75</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>RED</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="BLUE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>75</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>BLUE</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>101</y>
        <width>352</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Bit depth :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="binBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Binning :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bitBox"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Run and Save</string>
     </attribute>
     <widget class="QPushButton" name="SaveEBtn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>154</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
